jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -523,23 +529,19 @@ private boolean isDDLCommand(int updatedCount, int columnCount) throws SQLExcept
       }
 
       if (character == '\\') {
-        antiSlash = true;
+        backslash = true;
       }
 
       if (character == '\'') {
-        if (antiSlash) {
-          antiSlash = false;
-        } else if (quoteString) {
```

Comment:
this is supposed to fail, right? correct should be `select  '\''`, any thoughts?